pkg/apis/events/v1alpha1/channel_types.go

```diff
@@ -60,25 +59,21 @@ type ChannelSpec struct {
 	// Topic and Subscriptions. The value of the secret entry must be a
 	// service account key in the JSON format
 	// (see https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
+	// +optional
```

Comment:
we should probably document the default behaviour here?

Reply:
it is documented in the openapi spec, but this bit of code is not tied to the webhook code and it could change.